buzzmobile/process/frame_merger/frame_merger.py

```diff
@@ -1,6 +1,49 @@
+#!/usr/bin/env python
 import cv2
 import numpy as np
 
+import datetime as dt
+import rospy
+from cv_bridge import CvBridge, CvBridgeError
+from sensor_msgs.msg import Image
+
+
+class Frames(object):
```

Comment:
Can we rename this. Something like WorldModel? Not that important, but it is ambiguous.